ci/kokoro/docker/build.sh

```diff
@@ -33,7 +33,8 @@ elif [[ -n "${KOKORO_JOB_NAME:-}" ]]; then
   # files `$foo.cfg` for continuous builds and `$foo-presubmit.cfg` for
   # presubmit builds. Here we extract the value of "foo" and use it as the build
   # name.
-  export BUILD_NAME="$(basename "${KOKORO_JOB_NAME}" "-presubmit")"
+  BUILD_NAME="$(basename "${KOKORO_JOB_NAME}" "-presubmit")"
```

Comment:
Seems like you `git merge` the changes from `master`, in the future please `git rebase`, it is less confusing as I do not need to review other changes that are really no-ops.

Reply:
I did attempt a git rebase, but obviously did something wrong as it later made me merge.